x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -81,60 +77,55 @@ export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData
       width: '20%',
     },
     {
-      field: AnomalyDetectionListColumns.maxAnomalyScore,
       name: (
         <EuiToolTip
-          content={i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScoreTooltip', {
-            defaultMessage:
-              'Maximum score across all jobs in the group over its most recent 24 hour period',
+          content={i18n.translate('xpack.ml.overview.anomalyDetection.tableOverallScoreTooltip', {
+            defaultMessage: 'Overall anomaly scores within selected time range',
           })}
         >
           <span>
-            {i18n.translate('xpack.ml.overview.anomalyDetection.tableMaxScore', {
-              defaultMessage: 'Max anomaly score',
+            {i18n.translate('xpack.ml.overview.anomalyDetection.overallScore', {
+              defaultMessage: 'Overall score',
             })}
             <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
           </span>
         </EuiToolTip>
       ),
-      sortable: true,
-      render: (score: Group['max_anomaly_score']) => {
-        if (score === undefined) {
-          // score is not loaded yet
-          return <EuiLoadingSpinner />;
-        } else if (score === null) {
-          // an error occurred loading this group's score
-          return (
-            <EuiToolTip
-              content={i18n.translate(
-                'xpack.ml.overview.anomalyDetection.tableMaxScoreErrorTooltip',
-                {
-                  defaultMessage: 'There was a problem loading the maximum anomaly score',
-                }
-              )}
-            >
-              <EuiIcon type="alert" />
-            </EuiToolTip>
-          );
-        } else if (score === 0) {
-          return (
-            // @ts-ignore
-            <EuiHealth color={'transparent'} compressed="true">
-              {score}
-            </EuiHealth>
-          );
-        } else {
-          const color: string = getSeverityColor(score);
-          return (
-            // @ts-ignore
-            <EuiHealth color={color} compressed="true">
-              {getFormattedSeverityScore(score)}
-            </EuiHealth>
-          );
-        }
+      render: (group: Group) => {
```

Comment:
Can the contents of the Score column be center aligned (swim lane / No results found text)?

![image](https://user-images.githubusercontent.com/7405507/146040287-32c6d5aa-bf6a-44aa-89c8-236a3e5ca4f5.png)

Reply:
Fixed in eee71880cc4f8a469fd7c9ee0c6b67b425d4fe74